website/content/configuration/_index.md

```diff
@@ -4,28 +4,53 @@ weight: 4
 ---
 
 MetalLB remains idle until configured. This is accomplished by
-creating and deploying a config map into the same namespace
+creating and deploying various resources into the same namespace
 (metallb-system) as the deployment.
 
-There is an example config map in
-[`config/samples/example-config.yaml`](https://raw.githubusercontent.com/metallb/metallb/main/config/samples/example-config.yaml),
+There are various examples of the configuration CRs in
+[`config/samples`](https://raw.githubusercontent.com/metallb/metallb/main/config/samples),
 annotated with explanatory comments.
 
-If you've named the config map `config.yaml`, you can deploy the manifest with `kubectl apply -f config.yaml`.
+Also, the API is [fully documented here](../apis/_index.md).
 
-{{% notice note %}}
-If you installed MetalLB with Helm, you will need to change the
-namespace of the config map to match the namespace in which MetalLB was
-deployed, and change the name of the config map from `config` to
-`metallb`.
-{{% /notice %}}
+## Defining the IPs to assign to the Load Balancer services
+
+In order to assign an IP to the services, MetalLB must be instructed to do so via the
+`IPAddressPool` CR.
+
+All the IPs allocated via `IPAddressPool`s contribute to the pool of IPs that MetalLB
```

Comment:
nit: not sure if we should leave the s inside the block or outside :sweat_smile: is there some convention for this?

Reply:
Me neither, I just though `IPAddressPool` is the name of the object.